packages/fxa-payments-server/src/components/NewUserEmailForm/index.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import { Localized } from '@fluent/react';
+
+import shieldIcon from './images/shield.svg';
+
+import { Form, Input, Checkbox, OnValidateFunction } from '../fields';
+import {
+  State as ValidatorState,
+  useValidatorState,
+  MiddlewareReducer as ValidatorMiddlewareReducer,
+} from '../../lib/validator';
+
+import './index.scss';
+
+export type NewUserEmailFormProps = {
+  getString?: Function;
```

Comment:
Preferably we give this function a more specific type.  It sounds like it returns a string?  But just `Function` doesn't give the type system that.